org.testeditor.web.dropwizard.testing/src/main/java/org/testeditor/web/dropwizard/testing/files/FileTestUtils.xtend

```diff
@@ -0,0 +1,36 @@
+package org.testeditor.web.dropwizard.testing.files
+
+import com.google.common.io.Files
+import de.xtendutils.junit.AssertionHelper
+import java.io.File
+import javax.inject.Inject
+
+import static java.nio.charset.StandardCharsets.UTF_8
+
+class FileTestUtils {
+
+	@Inject protected extension AssertionHelper
+
+	def void assertFileExists(File parent, String path) {
+		val file = new File(parent, path)
+		file.exists.assertTrue('''Expected file does not exist: «file.absolutePath»''')
+	}
+
+	def void assertFileDoesNotExist(File parent, String path) {
+		val file = new File(parent, path)
+		file.exists.assertFalse('''Unexpected file found: «file.absolutePath»''')
+	}
+
+	def String read(File file) {
+		return Files.asCharSource(file, UTF_8).read
+	}
+
+	def void write(File targetDir, String path, String fileContents) {
```

Comment:
Naming of the parameters could be improved. The `File` is actually the `parent`, and the `path` could be named `fileName` or `child`. Instead of `fileContents` I would just use `contents?